bot.py

```diff
@@ -467,10 +468,28 @@ async def show_queue(self, message, _command_content):
 
         await message.channel.send(reply)
 
+    async def move(self, message, _command_content):
+        """
+        Moves the bot to the voice channel that the message author is currently
+        connected to.
+        """
+        if not await self.create_or_get_voice_client(message):
+            return
+        if message.author.voice.channel is self.voice_client.channel:
```

Comment:
I think `is` is correct here. We're referring to the same channel object.

Reply:
`author.voice.channel == self.voice_client.channel` does not imply `author.voice.channel is self.voice_client.channel`.
The docs only guarantee equality `==`.
https://discordpy.readthedocs.io/en/latest/api.html#discord.VoiceChannel

Reply:
```suggestion
        if message.author.voice.channel == self.voice_client.channel:
```

Reply:
Although now that I think about it, I do see a case where this might break:
If the voice_client.channel is instanciated by someone other than the person who is trying to call `!move`. 

I'll change `is` to `==`